optimize/components/userguide/instant-process-dashboards.md

```diff
@@ -4,28 +4,18 @@ title: Instant process dashboards
 description: "Use automatically generated dashboards and reports from Optimize to find insights for your processes."
 ---
 
-Camunda Optimize is a comprehensive process optimization tool that helps businesses streamline their operations
-and improve efficiency. One of the standout features of Optimize is its ability to automatically generate
-dashboards for each process, providing users with clear insights into process performance.
+Optimize serves as a comprehensive process optimization tool, streamlining operations and enhancing efficiency for businesses. Notably, one of its standout features is the automatic generation of dashboards for each process, offering users clear insights into process performance.
 
-Optimize imports each process deployed to Camunda and automatically creates a dashboard.
-The dashboards are designed to be intuitive and easy to use and can be accessed from the [process dashboards page](./process-dashboards.md) by clicking on the desired process.
+Upon importing each deployed process from Camunda, Optimize automatically creates an intuitive and user-friendly dashboard. Users can access these dashboards from the [process dashboards page](./process-dashboards.md) by selecting the desired process.
```

Comment:
Not sure if this is important, but this no longer happens during import. Instead, they are created on-demand for users when first accessed. The distinction does not impact users, so maybe no change required